java.d/src/main/java/org/firehol/netdata/plugin/configuration/exception/ConfigurationSchemeInstantiationException.java

```diff
@@ -29,4 +29,9 @@ public ConfigurationSchemeInstantiationException() {
 	public ConfigurationSchemeInstantiationException(String explanation) {
 		super(explanation);
 	}
+
+	public ConfigurationSchemeInstantiationException(String explanation, Throwable cause) {
+		this(explanation);
```

Comment:
Why not `super(explanation, cause)`?

Reply:
`javax.naming.ConfigurationException` doesn't seem to have a [constructor with that signature](https://docs.oracle.com/javase/8/docs/api/javax/naming/ConfigurationException.html#constructor.summary).